nxcomp/Log.h

```diff
@@ -198,18 +286,68 @@ NXLog& operator<< (NXLog& out, const NXLogStamp& value);
 
 
 template <typename T>
+bool has_newline(T value)
+{
+	return false;
+}
+
+template <char*>
+static bool has_newline(char *value)
+{
+	return strstr(value, "\n") != NULL;
```

Comment:
Can do, but do you think it's necessary? It seems like a slight performance impact that doesn't gain anything.